### PR TITLE
Tutorial/part two

### PR DESCRIPTION
**Pull request by dantereve**

### CODE REVIEW

File: docs/tutorial/part-two/index.md [hunk 1]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
+## Que contient ce tutoriel ?
 
-In this part, you're going to explore options for styling Gatsby websites and dive deeper into using React components for building sites.
+Dans cette partie, vous allez explorer les différentes options à votre disposition pour mettre en page un site Gatsby et vous irez plus loin dans l'utilisation des composants React pour créer des sites.
 
-## Using global styles
+## Utiliser des styles globaux
 
-Every site has some sort of global style. This includes things like the site's typography and background colors. These styles set the overall feel of the site — much like the color and texture of a wall sets the overall feel of a room.
+Tout site dispose d'une sorte de style de mise en page globale. Cela peut inclure des choses comme la typographie et les couleurs de fond. Ces styles servent à créer l'ambiance générale du site — une comme la couleur et la texture d'un mur donne une ambiance générale à une pièce.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
Tout site dispose d'une sorte de style de mise en page globale. Cela peut inclure des choses comme la typographie et les couleurs de fond. Ces styles servent à créer l'ambiance générale du site — comme la couleur et la texture d'un mur donne une ambiance générale à une pièce.
```

File: docs/tutorial/part-two/index.md [hunk 2]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-### Creating global styles with standard CSS files
+### Créer des styles globaux avec des fichiers CSS standards
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]: In the translation the reader may think that the CSS rules are standard which are not. I think the best translation of standard here should mean "regular" in French.

```suggestion
### Créer des styles globaux avec des fichiers CSS classiques
```

File: docs/tutorial/part-two/index.md [hunk 3]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-One of the most straightforward ways to add global styles to a site is using a global `.css` stylesheet.
+La manière la plus simple d'ajouter des styles globaux est d'utiliser une feuille de style globale en `.css`
 
-#### ✋ Create a new Gatsby site
+#### ✋ Créer un nouveau site Gatsby
 
-Start by creating a new Gatsby site. It may be best (especially if you're new to the command line) to close the terminal windows you used for [part one](/tutorial/part-one/) and start a new terminal session for part two.
+Commencez par créer un nouveau site Gatsby. Il est peut-être judicieux (particulièrement si vous êtes débutant en lignes de commandes) de fermer la fenêtre du terminal que vous avez utilisé pour la [première partie](/tutorial/part-one/) et démarrer une nouvelle instance de terminal.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
Commencez par créer un nouveau site Gatsby. Il est peut-être judicieux (particulièrement si vous êtes débutant dans l'utilisation des lignes de commandes) de fermer la fenêtre du terminal que vous avez utilisé pour la [première partie](/tutorial/part-one/) et démarrer une nouvelle instance du terminal.
```

File: docs/tutorial/part-two/index.md [hunk 4]
```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
...
 
-Open a new terminal window, create a new "hello world" gatsby site, and start the development server:
+Ouvrez une nouvelle fenêtre de terminal, créez un nouveau site "hello world", et lancez le serveur de développement:
 
 ```shell
 gatsby new tutorial-part-two https://github.com/gatsbyjs/gatsby-starter-hello-world
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]: Should we translate the code/command ? What do you think?

File: docs/tutorial/part-two/index.md [hunk 5]
```diff
@@ -48,9 +48,9 @@ You now have a new Gatsby site (based on the Gatsby "hello world" starter) with
 │       └── index.js
 ```
 
-#### ✋ Add styles to a css file
+#### ✋ Ajoutez des styles à votre fichier `.css`
 
-1. Create a `.css` file in your new project:
+1. Créez un fichier `.css` dans votre nouveau projet:
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
1. Créez un fichier `.css` dans votre nouveau projet :
```

File: docs/tutorial/part-two/index.md [hunk 6]
```diff
@@ -59,9 +59,9 @@ cd styles
 touch global.css
 ```
 
-> Note: Feel free to create these directories and files using your code editor, if you'd prefer.
+> Note: Vous êtes libre de créer ce dossier et ce fichier en utilisant votre éditeur de code si vous préférez.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
> Note : Vous êtes libre de créer ces dossiers et ces fichiers en utilisant votre éditeur de code si vous préférez.
```

File: docs/tutorial/part-two/index.md [hunk 7]
```diff
@@ -72,26 +72,26 @@ You should now have a structure like this:
...
+2. Définissez quelques styles dans le fichier `global.css` :
 
 ```css:title=src/styles/global.css
 html {
   background-color: lavenderblush;
 }
 ```
 
-> Note: The placement of the example css file in a `/src/styles/` folder is arbitrary.
+> Note: L'emplacement de ce fichier css prit en exemple `/src/styles/` est purement arbitraire.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
> Note : L'emplacement de ce fichier css d'exemple (dans le dossier`/src/styles/`) est purement arbitraire.
```

File: docs/tutorial/part-two/index.md [hunk 8]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
-> 💡 What is `gatsby-browser.js`? Don't worry about this too much and for now, just know that `gatsby-browser.js` is one of a handful of special files that Gatsby looks for and uses (if they exist). Here, the naming of the file **is** important. If you do want to explore more now, check out [the docs](/docs/browser-apis/).
+> 💡 Qu'est-ce que `gatsby-browser.js`? Ne vous préoccupez pas trop de ça pour l'instant. Sachez simplement que `gatsby-browser.js` est l'un des quelques fichiers spéciaux que Gatsby recherche et utilise (s'ils existent). Ici, le nom du fichier **est** important. Si vous voulez en apprendre davantage maintenant, jetez un oeil à [la documentation](/docs/browser-apis/).
 
-2. Import your recently-created stylesheet in the `gatsby-browser.js` file:
+2. Importez votre toute nouvelle feuille de style dans le fichier `gatsby-browser.js` :
 
 ```javascript:title=gatsby-browser.js
-import "./src/styles/global.css"
+import "./src/styles/global.css";
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]: Why do we add the semicolon?

File: docs/tutorial/part-two/index.md [hunk 9]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
 // or:
 // require('./src/styles/global.css')
 ```
 
-> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, we use `import` most of the time.
+> Note: Les deux syntaxes (`require`) de JS Vanilla et (`import`) des modules ES fonctionnent ici. Si vous n'êtes pas sûr de laquelle choisir, nous utilisons la plupart du temps `import`.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]: Actually, CommonJS is not a way to say "Javascript Commun" or pure JavaScript code (Vanilla JS) but it is [a real spec](http://www.commonjs.org/). We have to keep the name of the spec.

```suggestion
> Note : Les deux syntaxes (`require`) de CommonJS et (`import`) des modules ES fonctionnent ici. Si vous n'êtes pas sûr de laquelle choisir, nous utilisons la plupart du temps `import`.
```

File: docs/tutorial/part-two/index.md [hunk 10]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 ```
 
-If you take a look at your project in the browser, you should see a lavender background applied to the "hello world" starter:
+Si vous jetez un oeil à votre projet dans votre navigateur, vous devriez voir un fond de page couleur lavande dans votre kit de démarrage.
 
-![Lavender Hello World!](global-css.png)
+![Hello World Lavande!](global-css.png)
 
-> Tip: This part of the tutorial has focused on the quickest and most straightforward way to get started styling a Gatsby site — importing standard CSS files directly, using `gatsby-browser.js`. In most cases, the best way to add global styles is with a shared layout component. [Check out the docs](/docs/global-css/) for more on that approach.
+> Astuce: Cette partie du tutoriel s'est concentrée sur la façon la plus rapide et la plus simple de commencer à mettre en page un site Gatsby — en important une feuille de style classique dans `gatsby-browser.js`. Dans la plupart des cas, la meilleure façon de rajouter des styles globaux est d'utiliser un composant de mise en page partagé. [Regardez la documentation](/docs/global-css/) pour plus d'information sur cette méthode.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
> Astuce : Cette partie du tutoriel s'est concentrée sur la façon la plus rapide et la plus simple de commencer à mettre en page un site Gatsby — en important une feuille de style classique dans `gatsby-browser.js`. Dans la plupart des cas, la meilleure façon de rajouter des styles globaux est d'utiliser un composant de mise en page partagé. [Regardez la documentation](/docs/global-css/) pour plus d'information sur cette méthode.
```

File: docs/tutorial/part-two/index.md [hunk 11]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
-## Using component-scoped CSS
+## Utiliser du CSS lié à un composant
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
## Utiliser du CSS limité à un composant
```

File: docs/tutorial/part-two/index.md [hunk 12]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
-So far, we've talked about the more traditional approach of using standard css stylesheets. Now, we'll talk about various methods of modularizing CSS to tackle styling in a component-oriented way.
+Jusqu'à maintenant nous avons parlé de la méthode la plus traditionnelle en utilisant une feuille de style CSS standard. Maintenant, nous allons parler des diverses méthodes pour modulariser du CSS et appréhender les mises en page du point de vue des composants.
 
-### CSS Modules
+### Modules CSS
 
-Let's explore **CSS Modules**. Quoting from
-[the CSS Module homepage](https://github.com/css-modules/css-modules):
+À la découverte des ** Modules CSS**. Citation tirée de
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
À la découverte des **Modules CSS**. Citation tirée de
```

File: docs/tutorial/part-two/index.md [hunk 13]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
+> Astuce: Cette partie du tutoriel s'est concentrée sur la façon la plus rapide et la plus simple de commencer à mettre en page un site Gatsby — en important une feuille de style classique dans `gatsby-browser.js`. Dans la plupart des cas, la meilleure façon de rajouter des styles globaux est d'utiliser un composant de mise en page partagé. [Regardez la documentation](/docs/global-css/) pour plus d'information sur cette méthode.
 
-## Using component-scoped CSS
+## Utiliser du CSS lié à un composant
 
-So far, we've talked about the more traditional approach of using standard css stylesheets. Now, we'll talk about various methods of modularizing CSS to tackle styling in a component-oriented way.
+Jusqu'à maintenant nous avons parlé de la méthode la plus traditionnelle en utilisant une feuille de style CSS standard. Maintenant, nous allons parler des diverses méthodes pour modulariser du CSS et appréhender les mises en page du point de vue des composants.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
Jusqu'à maintenant nous avons parlé de la méthode la plus traditionnelle en utilisant une feuille de style CSS classique. Maintenant, nous allons parler des diverses méthodes pour modulariser du CSS et appréhender les mises en page du point de vue des composants.
```

File: docs/tutorial/part-two/index.md [hunk 14]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
-[the CSS Module homepage](https://github.com/css-modules/css-modules):
+À la découverte des ** Modules CSS**. Citation tirée de
+[la page d'accueil des modules CSS](https://github.com/css-modules/css-modules):
 
-> A **CSS Module** is a CSS file in which all class names and animation names
-> are scoped locally by default.
+> Un **Module CSS** est un fichier CSS où toutes les classes et animations ont une portée locale par défaut.
 
-CSS Modules are very popular because they let you write CSS normally but with a lot more safety. The tool automatically generates unique class and animation names, so you don't have to worry about selector name collisions.
+Les Modules CSS sont très populaires parce qu'ils vous permettent d'écrire du CSS normalement, mais avec plus sûreté. Cet outil génère automatiquement des noms de classes et d'animations uniques, de sorte que vous n'ayez pas à vous soucier de conflits dans vos sélecteurs.
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
Les Modules CSS sont très populaires parce qu'ils vous permettent d'écrire du CSS normalement, mais avec plus de sûreté. Cet outil génère automatiquement des noms de classes et d'animations uniques, de sorte que vous n'ayez pas à vous soucier de conflits dans vos sélecteurs.
```

File: docs/tutorial/part-two/index.md [hunk 15]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
-Gatsby works out of the box with CSS Modules. This approach is highly recommended for those new to building with Gatsby (and React in general).
+Gatsby fonctionne directement avec les Modules CSS. Cette approche est véritablement recommandée à tous ceux qui utilisent Gatsby depuis peu de temps (et React en général).
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
Gatsby fonctionne directement avec les Modules CSS. Cette approche est véritablement recommandée à tous ceux qui utilisent Gatsby (et React en général) depuis peu de temps.
```

File: docs/tutorial/part-two/index.md [hunk 16]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
-First, create a new `Container` component.
+Tout d'abord, créez un nouveau composant `Container`.
 
-1. Create a new directory at `src/components` and then, in this new directory, create a file named `container.js` and paste the following:
+1. Créez un nouveau dossier `src/components` et créez-y un fichier `container.js` en copiant le contenu suivant:
 
 ```javascript:title=src/components/container.js
-import React from "react"
-import containerStyles from "./container.module.css"
+import React from "react";
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
import React from "react"
```

File: docs/tutorial/part-two/index.md [hunk 17]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
+import containerStyles from "./container.module.css";
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
import containerStyles from "./container.module.css"
```

File: docs/tutorial/part-two/index.md [hunk 18]
```diff
@@ -103,67 +103,66 @@ Your project's file structure should now look like this:
...
 
 export default ({ children }) => (
   <div className={containerStyles.container}>{children}</div>
-)
+);
```
Comment (mindsers) [inline docs/tutorial/part-two/index.md]:
```suggestion
)
```

**Review by gatarelib — DISMISSED** [review]

**Review by mindsers — CHANGES_REQUESTED** [review]
Here some small changes w/ "suggestion" every time it is possible.

File: docs/tutorial/part-two/index.md (hunk 4 above)
Comment (dantereve) [reply docs/tutorial/part-two/index.md]: we could and maybe might translate `tutorial-part-two`

File: docs/tutorial/part-two/index.md (hunk 8 above)
Comment (dantereve) [reply docs/tutorial/part-two/index.md]: I've messed with prettier with no .prettierrc 😱

File: docs/tutorial/part-two/index.md (hunk 11 above)
Comment (dantereve) [reply docs/tutorial/part-two/index.md]: MUCH better 👌

File: docs/tutorial/part-two/index.md (hunk 4 above)
Comment (mindsers) [reply docs/tutorial/part-two/index.md]:
```suggestion
gatsby new tutoriel-partie-deux https://github.com/gatsbyjs/gatsby-starter-hello-world
```

**Review by mindsers — APPROVED** [review]

**Pull request merged**